source/OptimaJet.DWKit.StarterApplication/Repositories/ProjectRepository.cs

```diff
@@ -68,6 +68,11 @@ public override IQueryable<Project> Filter(IQueryable<Project> query, FilterQuer
             return base.Filter(query, filterQuery);
         }
 
+        public override IQueryable<Project> Sort(IQueryable<Project> entities, List<SortQuery> sortQueries)
```

Comment:
We may need to implement custom sorts later.
only sorting acceptance criteria was just "to sort", so currently sorting is only on the project name, and language